apps/addons/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^search/$', views.SearchView.as_view(), name='addons.api.search'),
+]
```

Comment:
So that's what we'll be doing now? Seems like a good idea (talking about the `api` subfolder in the apps, mirroring the current app split, which might need to be discussed some more in the (close?) future).